src/coloris.css

```diff
@@ -383,7 +383,7 @@ input.clr-color:focus {
 .clr-dark .clr-alpha div,
 .clr-dark .clr-swatches button,
 .clr-dark .clr-preview:before,
-.clr-dark .clr-field button {
+.clr-dark.clr-field button {
```

Comment:
@mdbassit how about allowing dark fields by adding the `clr-dark` class to them?

Reply:
Hmmm, it feels a bit messy, ideally we should use CSS custom properties (variables) to solve this, but that would require a slight rewrite of the CSS, I'd rather leave that for a later version. Once things settle a bit on my side, I'll look more into this along with your request for an inline picker.
For now, can you remove that selector, and we will get back to this later?

Reply:
Alright. Updated the PR. Yeah, I was also thinking variables would be a good idea.